hadoop-ozone/dist/src/main/smoketest/admincli/cert.robot

```diff
@@ -0,0 +1,39 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+*** Settings ***
+Documentation       Test ozone admin cert command
+Resource            ../commonlib.robot
+Test Timeout        5 minutes
+
+*** Keywords ***
+Setup Test
+    Run Keyword if    '${SECURITY_ENABLED}' == 'true'    Kinit test user     testuser     testuser.keytab
+
+*** Test Cases ***
+Run cert list
+    Pass Execution If       '${SECURITY_ENABLED}' == 'false'    N/A
+    ${output} =             Execute          ozone admin cert list
+    Should Contain          ${output}        Certificate list:(Type=
+
+cert list as JSON
```

Comment:
```suggestion
Cert list as JSON
```

Reply:
I'd suggest `List certificates` and `List certificates as JSON` (similar test case names are used in other `admincli` tests).

Reply:
I have updated it